data/json/items/comestibles/other.json

```
@@ -31,271 +31,349 @@
     "variants": [
       {
         "id": "gum_peppermint",
-        "name": { "str": "peppermint chewing gum" },
+        "name": { "str": "chewing gum, peppermint flavored" },
```

```suggestion
        "name": { "str_sp": "chewing gum, peppermint flavored" },
```
Needs plural handling

---

> Needs plural handling throughout

Thank you for letting me know, Procynae. I'll be doing it in no time.